### ISSUE TITLE
Symsorter panics on certain files

### ISSUE THREAD

**Issue by kristjanvalur** [getsentry/symbolicator#427]
### Environment
Now sorting linux .so files

### Steps to Reproduce

I'm sorting a bunch of cross-compiled .so files on windows

### Expected Result

Sorting works.

### Actual Result

symsorter reports panic:
```
libAkHarmonizer.so (lib, x86_64) -> symbols\36/41cb3edb33116a34711c08f36cd7ba/executable
03:32:42
  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ParseDebugIdError', crates\symsorter\src\app.rs:139:64
03:32:42
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
03:32:42
  thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: ParseDebugIdError', crates\symsorter\src\app.rs:139:64
03:32:42
  libAkToneGen.so (lib, x86_64) -> symbols\c6/a015089c25ad7b73b5ebfefb11ac66/executable
```

I will enable RUST_BACKTRACE=1.  I can also provide the abovementioned files in case of interest.
Should I continue reporting debug files that cause symsorter issues?

**Comment by Swatinem** [comment]
@kristjanvalur thanks for reporting this. This looks very similar to your report in #416 (same error message, although the other one looks garbled)

Which version (commit) are you using? Can you try with latest master (`1b9ba0530ad9ecc10dfddc06bb91dd9dc83cf48a`)?

I’m asking because we had master pointed to an experimental version of `symbolic` that caused the same error message, although with a completely different piece of code. And the file/line that your error message refers to has no unwraps related to that specific error type. All the unwraps are related to paths or locks.

**Comment by kristjanvalur** [comment]
So, confirmed that the problem is fixed in change d7e43cb9fbe1e92f8f7c3b3dc9bbed1329f7a7c7
I see there is much velocity in branch master.  Is there any particular branch I should be using for stability?

**Comment by jan-auer** [comment]
We're going to cut a release soon (cc @flub), it's probably best to use the tagged releases or published binaries.
